mm2src/coins/z_coin/z_coin_errors.rs

```diff
@@ -279,3 +279,29 @@ pub enum SpendableNotesError {
 
 #[derive(Debug, Display)]
 pub enum ZCoinBalanceError {}
+
+#[derive(Debug, Display)]
+pub enum ZcoinRpcError {
+    #[cfg(not(target_arch = "wasm32"))]
+    GrpcError(tonic::Status),
+    UtxoRpcError(UtxoRpcError),
+    JsonRpcError(JsonRpcError),
```

Comment:
These are not used, why have them here? I don't think there is a need for `ZcoinRpcError` you can just use `UpdateBlocksCacheErr::GrpcError`